### DISCUSSION TITLE
Add network_id in opentelekomcloud_vpc_subnet_v1

### DISCUSSION THREAD

**Discussion by r-ising** [opentelekomcloud/terraform-provider-opentelekomcloud#1008] [Q&A]
# Additional return value in opentelekomcloud_vpc_subnet_v1

Currently only the neutron_subnet_id is returned  as subnet_id in the terraform provider, but the api (https://docs.otc.t-systems.com/api/vpc/vpc_subnet01_0001.html) will return the network_id aswell. So it would be a nice feature if we can use the network_id other settings (firewall, etc..)

**Comment by outcatcher** [comment]
Resource `vpc_subnet_v1` attribute `id` is exactly openstack `network_id`:
`curl https://vpc.eu-de.otc.t-systems.com/v1/$PROJECT_ID/subnets/cfa0b4be-11eb-48d2-b59d-593daabc921f -H "X-Auth-Token: $TOKEN"`
```json
{
  "subnet": {
    "id": "cfa0b4be-11eb-48d2-b59d-593daabc921f",
    "name": "subnet-do-not-delete-pls",
    "description": "",
    "cidr": "192.168.0.0/16",
    "dnsList": [
      "100.125.4.25",
      "8.8.8.8"
    ],
    "status": "ACTIVE",
    "vpc_id": "de090f1e-cadb-4b3a-9d49-d413d7134af0",
    "ipv6_enable": false,
    "gateway_ip": "192.168.0.1",
    "dhcp_enable": true,
    "primary_dns": "100.125.4.25",
    "secondary_dns": "8.8.8.8",
    "neutron_network_id": "cfa0b4be-11eb-48d2-b59d-593daabc921f",
    "neutron_subnet_id": "89ddc90e-7dad-4904-b364-1cddc7018608",
    "extra_dhcp_opts": []
  }
}
```

**Comment by outcatcher** [comment]
This is also described in the resource documentation:
https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs/resources/vpc_subnet_v1#attributes-reference

**Comment by outcatcher** [comment]
After further discussion will be implemented in #1014
